lib/ansible/modules/net_tools/nios/nios_txt_record.py

```diff
@@ -43,6 +43,7 @@
         per substring, up to a total of 512 bytes. To enter leading,
         trailing, or embedded spaces in the text, add quotes around the
         text to preserve the spaces.
+    required: true
```

Comment:
can u put a Note, why the param has been changed to required param

Reply:
This change was made to fix #62377